eslint-config/resolved-configs/list-plugins.js

```diff
@@ -0,0 +1,135 @@
+const fs = require('node:fs/promises');
+const path = require('node:path');
+
+async function getFilesRecursively(inputDirectory, result = []) {
+  const directoryEntries = await fs.readdir(inputDirectory);
+  for (const directoryEntry of directoryEntries) {
+    const filePath = path.join(inputDirectory, directoryEntry);
+    if ((await fs.stat(filePath)).isDirectory()) {
+      await getFilesRecursively(filePath, result);
+    } else {
+      result.push(`./${filePath}`);
+    }
+  }
+
+  return result;
+}
+
+function addPluginPrefix(pluginName) {
+  const eslintPluginPackagePrefix = 'eslint-plugin';
+  if (pluginName.startsWith(eslintPluginPackagePrefix)) {
+    return pluginName;
+  }
+  return `${eslintPluginPackagePrefix}-${pluginName}`;
+}
+
+function getPackageName(rawPlugin) {
+  const packageNameOverwriteMap = new Map();
+  packageNameOverwriteMap.set('eslint-plugin-vitest', '@vitest/eslint-plugin');
+
+  if (rawPlugin.includes(':')) {
+    const packageDefinition = rawPlugin.split(':')[1];
+    const parts = packageDefinition.split('@');
+    const pluginName = parts[0];
+
+    if (pluginName === '') {
+      parts.pop();
+      return parts.join('@');
+    }
+    const packageName = addPluginPrefix(pluginName);
+    return packageNameOverwriteMap.get(packageName) ?? packageName;
+  }
+  const packageName = addPluginPrefix(rawPlugin);
+  return packageNameOverwriteMap.get(packageName) ?? packageName;
+}
+
+function getAllNpmPackages() {
+  const lockJsonPackages = Object.entries(
+    // eslint-disable-next-line global-require
+    require('../package-lock.json').packages
+  );
+  const lockJsonPackagesMap = new Map();
+  for (const [packageName, packageManifest] of lockJsonPackages) {
+    lockJsonPackagesMap.set(
+      packageName.replaceAll('node_modules/', ''),
+      packageManifest
+    );
+  }
+  return lockJsonPackagesMap;
+}
+
+function deduplicateArray(items) {
+  return Array.from(new Set(items));
+}
+
+async function updateReadmePluginSection(pluginSectionContents) {
```

Comment:
Doc: This should maybe look for specific markdown comment to insert the list in, because if you only read the `README.md` code you won't know that it is generated automatically here is a reference on how I achieved something similar by copying `eslint-doc-generator`
(Alternatively you can just put a markdown comment in the `README.md` to indicate that the list was autogenerated):
https://github.com/CloudNStoyan/arabasta/blob/aec05290725bdb87452d07f1efefdeccadfb066d/eslint-plugin-api-documentation/.eslint-doc-generatorrc.js#L11
https://github.com/CloudNStoyan/arabasta/blob/aec05290725bdb87452d07f1efefdeccadfb066d/eslint-plugin-api-documentation/.eslint-doc-generatorrc.js#L61-L84

Here is how it look in the markdown file:
https://github.com/CloudNStoyan/arabasta/blob/aec05290725bdb87452d07f1efefdeccadfb066d/eslint-plugin-api-documentation/docs/rules/require-jsdoc-returns.md?plain=1#L17-L36

Reply:
Done.